tests/test_utils.py

```diff
@@ -1034,6 +1041,11 @@ def test_intersect(objs, expected):
         audformat.utils.intersect(objs),
         expected,
     )
+    objs = list(reversed(objs))
+    pd.testing.assert_index_equal(
+        audformat.utils.intersect(objs),
+        expected,
+    )
```

Comment:
I would remove this part and instead test all possible permutations by updating the first part of the code to:

```python
    for permuted_objs in itertools.permutations(objs):
        pd.testing.assert_index_equal(
            audformat.utils.intersect(list(permuted_objs)),
            expected,
        )
```

`list()` is only needed until we have fixed to support `Sequence` instead of `List` as input as you are doing in https://github.com/audeering/audformat/pull/229.

Reply:
I created https://github.com/audeering/audformat/pull/235 that applies the fix.

Reply:
Thanks, it's merged now.

Reply:
I applied the fix and changed to permutations.